lib/assets/javascripts/twine.coffee

```diff
@@ -69,11 +69,16 @@
       element = findOrCreateElementForNode(node)
       element.indexes = indexes
 
-    for type, binding of Twine.bindingTypes when definition = Twine.getAttribute(node, type)
-      element = findOrCreateElementForNode(node)
-      element.bindings ?= []
-      element.indexes ?= indexes
-
+    element = findOrCreateElementForNode(node)
+    element.bindings ?= []
+    element.indexes ?= indexes
+    Array.prototype.slice.call(node.attributes).forEach (attribute) ->
```

Comment:
I would expect a for loop to be fine. Alternatively `[].forEach.call` by I still don't see why.